### PR TITLE
Amend delete command

### PR DESCRIPTION
**Pull request by NicolasCwy**
Amend delete command to delete a guest and a staff model

### CODE REVIEW

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -53,7 +53,7 @@
     ReadOnlyAddressBook getAddressBook();
 
     /**
-     * Returns true if a person with the same identity as {@code person} exists in the address book.
+     *      * Returns true if a person with the same identity as {@code person} exists in the address book.
```
Comment (calvintanwj) [inline src/main/java/seedu/address/model/Model.java:56]: Looks like you accidentally added a asterisk here.

**Review by calvintanwj — COMMENTED** [review]
Just that one hiccup to change.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -109,7 +109,7 @@ Example 2:
 <br>![StaffAddDiagram](images/StaffAddDiagram.png)
 
 * `list` command lists all contact details of people in the address book.
-* `add n/Jeremy sid/321 p/87655432` , adds a new staff, Jeremy to PH and shows the new
+* `add n/Jeremy sid/321 p/87655432` , adds a new staff, Jeremy to **PH** and shows the new
```
Comment (alyssa-savier) [inline docs/UserGuide.md:112]: the error is not here, but for the part on 'delete' the diagram shows 321 and the text command shows 123.
'delete sid/123, The staff, Thomas The Train, who has the staff ID 123, is deleted from PH.'

Comment (rgbpokka) [reply docs/UserGuide.md:112]: The new change looks correct to me. The staff id (321) matches the one on the diagram (321). Am I missing something?

**Pull request merged**